.travis.yml

```diff
@@ -1,15 +1,36 @@
+sudo: required
+dist: trusty
 language: php
+
+matrix:
+  include:
+   #5.3.1 Ubuntu Precise exceptions (not 5.3.3 since there is no SSL support there)
+   # Fedora 3.5 has a bug that makes tests fail. we should deprecate it fully
+    #- php: 5.3.3
+    #  dist: precise
+    #  env: FEDORA_VERSION="3.5"
+    - php: 5.3.1
```

Comment:
```yaml
matrix:
  include:
    - php: 5.3
      dist: precise
      env: FEDORA_VERSION="3.6.2"
    - php: 5.3
      dist: precise
      env: FEDORA_VERSION="3.7.0"
    - php: 5.3
      dist: precise
      env: FEDORA_VERSION="3.8.1"
```

If we just use vanilla 5.3 it should work like the snippet above.

Reply:
My fault. Tried to be smarter than travis-ci since they where pretty specific in their website about 5.3.10 fixing now